Traceback (most recent call last):
  File "/Users/hutianzheng/Desktop/studying/Y2-P3-ml4grpahs/DGI_CODE/ML4Graphs_DGI/main.py", line 88, in <module>
    model.load_state_dict(torch.load('best_dgi2.pkl'))
  File "/opt/anaconda3/envs/DeepL/lib/python3.8/site-packages/torch/nn/modules/module.py", line 2152, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for DGI:
	Missing key(s) in state_dict: "gcnlayer.bias", "gcnlayer.fc.weight", "gcnlayer.act.weight", "discriminator.score.weight", "discriminator.score.bias".
	Unexpected key(s) in state_dict: "gcn.bias", "gcn.fc.weight", "gcn.act.weight", "disc.f_k.weight", "disc.f_k.bias".